### PR TITLE
Update coding-style.md

### PR DESCRIPTION
**Pull request by dlandi**
I have added rule #18 which is Aprogenia's "Blank Line After Curly Brace" rule.

Note: I did not have to modify the sample code as it already observed that rule.

### CODE REVIEW

File: Documentation/coding-style.md [hunk 1]
```diff
@@ -32,6 +32,30 @@ The general rules:
 `public int X { get; private set; }`  
 17. Avoid initializing instance properties outside of constructor.  
 `public int MaxItems { get; set; } = 100000;`  
+18. Always provide a blank line after a curly brace that is not followed by a another curly brace.
```
Comment (fassadlr) [inline Documentation/coding-style.md]: Extra "a" there ;)

Comment (fassadlr) [inline Documentation/coding-style.md]: Extra "a" there ;)

**Comment by bokobza** [comment]
Let's also add these to the rules (these are stylecop rules 1505, 1507, 1508 and 1509 which I can add to the settings):

- Opening curly brackets must not be preceded or followed by blank line
- Closing curly brackets must not be preceded by blank line
- Code must not contain multiple blank lines in a row

thoughts?

**Comment by dangershony** [comment]
Yes lets also add the style cop additions as well

**Comment by bokobza** [comment]
@dlandi could you please amend the file to include my comment above? Also, tell me if you disagree, in which case we can discuss.

**Comment by dlandi** [comment]
@bokobza Sure.  I misunderstood your message.  I'm on it.

**Comment by dangershony** [comment]
Looks good to me, I will push tomorrow if no objections.

**Comment by bokobza** [comment]
All good, thanks @dlandi !

**Comment by bokobza** [comment]
Nope, it's not letting me merge it.
@fassadlr I think only you haz the powa!!!!

**Pull request merged**